operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -512,6 +543,26 @@ protected Map<String, CertAndKey> maybeCopyOrGenerateCerts(
         return certs;
     }
 
+    private boolean isNewVersion(Secret secret, String podName) {
```

Comment:
Can you add some Javadoc here explaining what it does and why is it named like this? Without the comment on line 497, it is completely unclear what version it is talking about in the method name. Only the comment on line 497 actually makes it a bit clear.

Something like _Secrets created by older operator versions did not contain all the fields. This method checks if they are present._ or at least something simple like that.

Reply:
Done